cumulus/thin-egress.tf

```diff
@@ -64,20 +64,38 @@ resource "aws_cloudwatch_log_subscription_filter" "egress_api_gateway_log_subscr
 }
 
 # Egress Lambda Log Group
+# does it already exist
+data "aws_cloudwatch_log_group" "egress_lambda_log_group" {
+  name = "/aws/lambda/${module.thin_egress_app.egress_lambda_name}"
+}
```

Comment:
Maybe it's possible to use an `import` block from terraform 1.5 here? https://developer.hashicorp.com/terraform/language/import

I've never used these before so I have no idea if they are meant to be used for this sort of situation or would work well, but perhaps it's a way to avoid having both a `data` and a `resource` object for the same cloudwatch group.

Reply:
Might be able to avoid having to define 2 filter resources that way.

Reply:
I just created my own kinesis stream and log put destination in my sandbox account per these instructions so I could emulate the cloud metrics subscription filter.

https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CreateDestination.html

I then deleted the `my-deployment-thin-egress-app-EgressLambda` cloudwatch group and ran the code.

got this error:

```
Error: reading CloudWatch Logs Log Group (/aws/lambda/cxl-cumulus-cxl-thin-egress-app-EgressLambda): empty result

  with data.aws_cloudwatch_log_group.egress_lambda_log_group,
  on thin-egress.tf line 68, in data "aws_cloudwatch_log_group" "egress_lambda_log_group":
  68: data "aws_cloudwatch_log_group" "egress_lambda_log_group" {
```

So the code as-written doesn't work if the cloudwatch group doesn't exist.